specification/2.0/README.md

```diff
@@ -4074,7 +4074,7 @@ Taking advantage of the fact that `roughness` is shared between metal and dielec
 const dielectricSpecular = 0.04
```

Comment:
The constant `dielectricSpecular` could be removed, it's not used anymore.

Reply:
Done.